components/Player/Player.tsx

```diff
@@ -384,28 +387,33 @@ const Player: React.FC<IPlayerProps> = ({
   );
 
   useEffect(() => {
+    const audioElmTemp = audioElm;
+
     // Initialize audio element.
     if (!audioElm.current) {
-      audioElm.current = new Audio();
+      // audioElm.current = new Audio();
```

Comment:
What's this about?  Seems like a no-op `if`.

Reply:
Oh, this is old code. Originally the player used an in-memory audio elements. Had to switch to rendering an audio element with track elements. Couldn't find a good way to add tracks to the in-memory audio element that was less complicated than just rendering a track element. This can be removed.